src/transformers/trainer.py

```diff
@@ -902,13 +929,18 @@ def _save_checkpoint(self, model, trial, metrics=None):
         if self.is_world_process_zero():
             self._rotate_checkpoints(use_mtime=True)
 
+    def _run_name(self, trial):
```

Comment:
This seems to be only used once, not sure we need the refactor in a separate function for just two lines?

Reply:
Solved in https://github.com/huggingface/transformers/pull/7935/commits/d83d80d075b5dd7278020c93a3fb0235a3405ecc